cmd/registry/patterns/resources.go

```diff
@@ -52,6 +54,10 @@ func (s SpecName) Spec() string {
 	return s.Name.String()
 }
 
+func (s SpecName) Revision() string {
```

Comment:
This method can be unexported because it won't be accessible from the `ResourceName` interface.

Reply:
ok